spinalcordtoolbox/deepseg/models.py

```diff
@@ -183,7 +185,7 @@
          "thr": None,  # Images are already binarized
          "default": False,
      },
-    "model_seg_ms_lesion": {
+      "model_seg_ms_lesion": {
```

Comment:
The Code Quality check failed with the following message:

```
spinalcordtoolbox/deepseg/models.py:188:7: E131 continuation line unaligned for hanging indent
```

This one is my fault, I added this in my merge commit. My bad! :sweat: 

```suggestion
    "model_seg_ms_lesion": {
```